pages/infrastructure_providers-validators/how_to_set_up_full_node.md

```diff
@@ -1,70 +1,131 @@
-```bash
-DYDX_HOME=/home/vmware/.dydx-mainnet-1
-
-nohup dydxprotocold start --p2p.seeds="ade4d8bc8cbe014af6ebdf3cb7b1e9ad36f412c0@seeds.polkachu.com:23856,65b740ee326c9260c30af1f044e9cda63c73f7c1@seeds.kingnodes.net:23856,f04a77b92d0d86725cdb2d6b7a7eb0eda8c27089@dydx-mainnet-seed.bwarelabs.com:36656,20e1000e88125698264454a884812746c2eb4807@seeds.lavenderfive.com:23856,c2c2fcb5e6e4755e06b83b499aff93e97282f8e8@tenderseed.ccvalidators.com:26401,4f20c3e303c9515051b6276aeb89c0b88ee79f8f@seed.dydx.cros-nest.com:26656,a9cae4047d5c34772442322b10ef5600d8e54900@dydx-mainnet-seednode.allthatnode.com:26656,802607c6db8148b0c68c8a9ec1a86fd3ba606af6@64.227.38.88:26656,4c30c8a95e26b07b249813b677caab28bf0c54eb@rpc.dydx.nodestake.top:666,ebc272824924ea1a27ea3183dd0b9ba713494f83@dydx-mainnet-seed.autostake.com:27366" --home=$DYDX_HOME --non-validating-full-node=true > /tmp/fullnode.log 2>&1 &
-```
-2. You can tail the log to see the progress.
-```bash
-tail -f /tmp/fullnode.log
-```
-3. The full node is now syncing. To determine whether the full node is caught up with the chain head, please check the applicable block explorer to determine when it reaches the current block – an example block explorer is shown on https://www.mintscan.io/dydx
-
-## Things you can do with the full node
-GET CURRENT BLOCK: You can get the current block with this program https://github.com/chiwalfrm/dydxexamples/blob/1d46b7a75499205d9c1c1986ae4ae8f21b6c1385/v4block_subscribe.py
-
-Run it with the full node IP address and port `26657`:
-```bash
-python3 v4block_subscribe.py ws://<IPADDRESS>:26657
-```
-Where `<IPADDRESS>` is the IP address of your full node.
-![Full node usage example](../../artifacts/how_to_set_up_full_node_usage_example.png)
\ No newline at end of file
+# Set Up a Full Node
+
+> Examples on this page are for mainnet deployments by dYdX token holders. For information on alternative deployment types, including testnet deployments in the United States, see the [Network Constants page](../infrastructure_providers-network/network_constants.mdx).
+
+## Prerequisites
+- Linux (Ubuntu Server 22.04.3 or later recommended)
+- 8-core CPU (ARM or x86_64)
+- 64 GB RAM
+- 500 GB SSD NVME Storage
+
+## Download the `dydxprotocold` binary and initialize the data directory
+The binary contains the software you need to operate a full node. Initializing a data directory dedicates a folder on your system to recordkeeping, such as logging.
+
+1. Find the latest `Release protocol` from the [v4 Chain Releases](https://github.com/dydxprotocol/v4-chain/releases/) page. Download the compressed source code file appropriate for your system.
+   
+   For example, to use `Release protocol/v5.0.5` on Ubuntu, download `Source code.tar.gz`.
```

Comment:
Hmm this might be off. We want them to download the `dydxprotocold-xyz.tar.gz` files ([example](https://github.com/dydxprotocol/v4-chain/releases/tag/protocol%2Fv5.0.5)) rather than the source code